api/src/repositories/survey-location-repository.ts

```diff
@@ -0,0 +1,96 @@
+import SQL from 'sql-template-strings';
+import { z } from 'zod';
+import { PostLocationData } from '../models/survey-create';
+import { PutSurveyLocationData } from '../models/survey-update';
+import { generateGeometryCollectionSQL } from '../utils/spatial-utils';
+import { BaseRepository } from './base-repository';
+
+export const SurveyLocationRecord = z.object({
+  survey_location_id: z.number(),
+  name: z.string(),
+  description: z.string(),
+  geography: z.any(),
+  geojson: z.any(),
```

Comment:
Could we make this `z.object` to be a little more specific? Are we going with `z.any` because this can be a JSON string _or_ JSON object?

Reply:
I think it's beneficial to answer the question "is this a JSON string or is this a JSON object" as early on in the request as possible

Reply:
`geojson` is currently always an array of Feature, I believe.

I've updated these definitions to be a bit more specific.